sdks/java/io/iceberg/src/main/java/org/apache/beam/sdk/io/iceberg/RecordWriter.java

```diff
@@ -17,15 +17,12 @@
  */
 package org.apache.beam.sdk.io.iceberg;
 
-import static org.apache.beam.sdk.io.iceberg.IcebergUtils.beamRowToIcebergRecord;
```

Comment:
can we add this to CHANGES.md to call it out?

Reply:
done